lib/SPIRV/OCLToSPIRV.cpp

```diff
@@ -179,9 +179,6 @@ bool OCLToSPIRVBase::runOCLToSPIRV(Module &Module) {
   for (auto &I : ValuesToDelete)
```

Comment:
Perhaps we should change the type of `ValuesToDelete` to `std::set<Instruction *>` and avoid inserting the called functions into it?

Reply:
done, thank you